### PR TITLE
refactor(cdk/testing): make the protractor environment a thin wrapper around the webdriver environment

### PR DESCRIPTION
**Pull request by mmalerba**

### CODE REVIEW

**Comment by devversion** [comment]
@googlebot I consent.

File: tools/server-test/test-runner.ts
Comment (devversion) [inline tools/server-test/test-runner.ts:1]: For a follow-up: can we add a small top-level comment that explains what is done as part of this runner?

File: src/cdk/testing/protractor/protractor-element.ts
Comment (devversion) [inline src/cdk/testing/protractor/protractor-element.ts:15]: It looks like the e2e tests fail because there is no stabilize function, and we no longer go through the Protractor patched functions (that rely on the `blocking-proxy`). Should we just use the stabilizers as we do for plain webdriver?

Comment (devversion) [reply src/cdk/testing/protractor/protractor-element.ts:15]: Alternatively, we might be able to just use the Protractor `browser.waitForAngular` function?
https://github.com/angular/protractor/blob/df3dd6400e3a4792ff9f7a58849d2a5eed6e1483/lib/browser.ts#L620

**Review by devversion — APPROVED** [review]
LGTM. Just a few minor comments

File: src/cdk/testing/protractor/protractor-element.ts
Comment (mmalerba) [reply src/cdk/testing/protractor/protractor-element.ts]: Hmm, oddly passing the `waitForAngular` function didn't seem to have any effect

**Comment by mmalerba** [comment]
I'm going to hold off on this until after the webdriver environment lands on master branch

**Pull request closed without merging**
